src/core/hooking/Hooking.cpp

```diff
@@ -5,17 +5,18 @@
 #include "VMTHook.hpp"
 #include "game/hooks/Hooks.hpp"
 #include "game/pointers/Pointers.hpp"
+#include "../memory/ModuleMgr.hpp"
```

Comment:
Relative include instead of include from the root of the project.

Reply:
Relative fails for me